src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -53,11 +53,16 @@
 @Description({"Writes text into a .log file. Skript will write these files to /plugins/Skript/logs.",
 		"NB: Using 'server.log' as the log file will write to the default server log. Omitting the log file altogether will log the message as '[Skript] [&lt;script&gt;.sk] &lt;message&gt;' in the server log."})
 @Examples({"on place of TNT:",
-		"	log \"%player% placed TNT in %world% at %location of block%\" to \"tnt/placement.log\""})
-@Since("2.0")
+	    "	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"",
+	    "	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with severity of info\"",
+		"	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with severity of warning\"",
+        "   log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with severity of error\""})
```

Comment:
Was gonna suggest but this needs fixed as well, reformat with tabs!!! don't mix tabs and spaces  before the ".
Additionally use `\t` instead of a tab/space also will need to indent 1 line
```
"on blah:",
	"\tblah blah"
```

Reply:
Ahh alright!

Reply:
Fixed!